docs/external-node.md

```diff
@@ -266,13 +266,24 @@ please refer to the [getting-started guide](getting-started.md#open-vswitch).
    ```
 
 2. The `antrea-agent.conf` file specifies agent configuration parameters. Copy
-   the [agent configuration file](../build/yamls/externalnode/conf/antrea-agent.conf)
-   to the VM and edit the `antrea-agent.conf` file to set `clientConnection`,
-   `antreaClientConnection` and `externalNodeNamespace` with the correct values.
-   Copy `antrea-agent.antrea.kubeconfig` and `antrea-agent.kubeconfig` files to
+   the [agent configuration file](../build/yamls/externalnode/conf/antrea-agent.conf),
+   and `antrea-agent.antrea.kubeconfig` and `antrea-agent.kubeconfig` files to
    the VM, that were generated in the step 4 and step 5 of
    [Prerequisites on Kubernetes cluster](#prerequisites-on-kubernetes-cluster).
 
+3. Bootstrap `antrea-agent` using the [installation script](../hack/externalnode/install-vm.sh)
+   as shown below and skip the remaining steps.
+
+    ```bash
+    ./install-vm.sh --ns vm-ns --bin ./antrea-agent --config ./antrea-agent.conf \
+    --kubeconfig ./antrea-agent.kubeconfig \
+    --antrea-kubeconfig ./antrea-agent.antrea.kubeconfig --nodename vm1
+    ```
+
+4. Bootstrap `antrea-agent` manually by editing the `antrea-agent.conf` file to
```

Comment:
3 or 4-5 are two different ways to bootstrap `antrea-agent`? If so maybe we should list them as two options under "3. Bootstrap `antrea-agent`"?

Reply:
yes, there are 2 ways to bootstreap agent.  If the existing doc does not look good, I can change it to have 2 options.

3. Bootstrap `antrea-agent`
3.1  Auto bootstrap `antrea-agent`
3.2  Manual bootstrap `antrea-agent`
-  
-

Reply:
Yes two sub-options looks better to me.

Reply:
Updated it. Please help with review.